system/Database/Query.php

```diff
@@ -315,13 +315,13 @@ public function getOriginalQuery(): string
 	 *
 	 * @return void
 	 *
-	 * @see https://regex101.com/r/EUEhay/1 Test
+	 * @see https://regex101.com/r/qvzPaZ/1
```

Comment:
can use versioned regex101 uri: 

```suggestion
       * @see https://regex101.com/r/EUEhay/3
```